### PR TITLE
reformatting bitwise_invert

### LINKED ISSUE #7916: bitwise_invert

### PR DESCRIPTION
**Pull request by AhmedEssam19**
Resolves #7916

### CODE REVIEW

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Hey! 
Your PR looks amazing ✨!
Just requested one change that is mentioned below which needs to be addressed before we can merge your PR! 🙂

1. You've added the container instance method docstrings kindly add the container static method docstrings as well.

Thanks!

**Comment by AhmedEssam19** [comment]
@zaeemansari70 Thanks for your feedback. I have made the changes. kindly review them and let me know if there are other needed modifications.

File: ivy/container/elementwise.py [hunk 1]
```diff
@@ -1485,7 +1485,7 @@ def bitwise_left_shift(
 
     @staticmethod
     def static_bitwise_invert(
-        x: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        x: ivy.Container,
```
Comment (zaeemansari70) [inline ivy/container/elementwise.py]: This shouldn't have been changed in the process, only the docstrings were to be added as the typehints are correct, we take in `ivy.Array and ivy.NativeArray` as well in the static method . 🙂

**Review by zaeemansari70 — APPROVED** [review]
Lgtm!

File: ivy/container/elementwise.py (hunk 1 above)
Comment (AhmedEssam19) [reply ivy/container/elementwise.py]: @zaeemansari70 I have tried this method out previously and it didn't work with arrays or native arrays, only with containers. That's why I have changed the type hints

**Pull request merged**